src/main/helm/helm-chart-manager.ts

```diff
@@ -20,86 +20,100 @@
  */
 
 import fs from "fs";
+import v8 from "v8";
 import * as yaml from "js-yaml";
-import { HelmRepo, HelmRepoManager } from "./helm-repo-manager";
+import type { HelmRepo } from "./helm-repo-manager";
 import logger from "../logger";
 import { promiseExec } from "../promise-exec";
 import { helmCli } from "./helm-cli";
 import type { RepoHelmChartList } from "../../common/k8s-api/endpoints/helm-charts.api";
-
-type CachedYaml = {
-  entries: RepoHelmChartList
-};
+import { sortCharts } from "../../common/utils";
 
 export class HelmChartManager {
-  protected cache: any = {};
-  protected repo: HelmRepo;
+  static #cache = new Map<string, Buffer>();
```

Comment:
We're not using such style anywhere else. Probably better to omit the hash sign?

Reply:
This is a better version of `private` as it is actually enforced at runtime

Reply:
Why is this better? And why mark `static` property as private? Asking because static property can not be called on instance of a class.

Reply:
Same reason to mark an instance property as `private`, so to better model the encapsulation.

A `static` property that is `#` (JS private) cannot be accessed by code that is not a static member or an instance member.